changelog/fix_false_negative_for_useless_assignment.md

```diff
@@ -0,0 +1 @@
+* [#13309](https://github.com/rubocop/rubocop/issues/13309): Fix and spec false negative for `Lint/UselessAssignment` cop when there is an useless assignment followed by a block. ([@pCosta99][])
```

Comment:
Can you update it to the following text?

```suggestion
* [#13309](https://github.com/rubocop/rubocop/issues/13309): Fix a false negative for `Lint/UselessAssignment` cop when there is a useless assignment followed by a block. ([@pCosta99][])
```

Reply:
Done! :+1: